fuse/dl/models/backbones/backbone_transformer.py

```diff
@@ -45,3 +47,119 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
         x = self.dropout(x)
         x = self.transformer(x)
         return x
+
+
+class CrossAttentionTransformer(nn.Module):
+    """
+    CrossAttentionTransformer backbone model based on x-transformers library.
+
```

Comment:
suggestion: 
maybe add some link to a paper/blogpost on the component?

Reply:
@avihu111 
Good idea , thanks :)